pyproject.toml

```diff
@@ -55,7 +55,7 @@ packages = ["uncertainties", "uncertainties.unumpy"]
 [tool.setuptools_scm]
 
 [project.urls]
-Documentation = "http://uncertainties-python-package.readthedocs.io/"
+Documentation = "http://uncertainties.readthedocs.io/"
```

Comment:
this could/should use “https”

Reply:
@reneeotten  Thanks - good catch and will fix.